target/linux/ramips/image/mt7620.mk

```diff
@@ -340,6 +340,18 @@ define Device/edimax_ew-7478apc
 endef
 TARGET_DEVICES += edimax_ew-7478apc
 
+define Device/elecom_wrc-733ghbk
+  MTK_SOC := mt7620a
```

Comment:
Rename to "SOC" due to 58e71e322bc0e0edd08157075e9022c15ae33ec5.

Reply:
done